src/orchestration/Orchestration.ts

```diff
@@ -276,6 +278,16 @@ export class Orchestration {
         this.dispatchManager.setAwsCredentials(credentials);
     }
 
+    /**
+     * Set custom session attributes to add them to all event metadata.
+     * @param payload object containing custom attribute data in the form of key, value pairs
+     */
+    public setCustomAttributes(payload: {
```

Comment:
nit: `setSessionAttributes` to be consistent with the config option + rename parameter
```suggestion
    public setSessionAttributes(attributes: {
```